docs/src/api/class-frame.md

```diff
@@ -744,6 +744,8 @@ When all steps combined have not finished during the specified [`option: timeout
 ### option: Frame.hover.timeout = %%-input-timeout-%%
 
 ## async method: Frame.innerHTML
+* langs:
+  - alias-csharp: GetInnerHTMLAsync
```

Comment:
I don't like this. It means we need to add this across the entire docs and make sure to keep consistent in the future. If we can already say _it has to be this way_, we can force the `Get` prefix for async methods without parameters in the generator.